src/core/api/index.ts

```diff
@@ -1193,8 +1194,9 @@ class Player extends EventEmitter<PLAYER_EVENT_STRINGS, any> {
       return [];
     }
 
-    return videoAdaptation.representations
+    const bitrates = videoAdaptation.representations
```

Comment:
Now that I'm thinking about it, why not use `videoAdaptation.getAvailableBitrates()` and uniq that one instead.